core/services/keystore/keys/tonkey/export.go

```diff
@@ -0,0 +1,49 @@
+package tonkey
+
+import (
+	"encoding/hex"
+
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+
+	"github.com/smartcontractkit/chainlink/v2/core/services/keystore/internal"
+	"github.com/smartcontractkit/chainlink/v2/core/utils"
+)
+
+const keyTypeIdentifier = "Ton"
```

Comment:
Should this be canonical "TON"?

```suggestion
const keyTypeIdentifier = "TON"
```